package.json

```diff
@@ -153,7 +152,7 @@
     "lookup-closest-locale": "6.0.4",
     "metaviewport-parser": "0.2.0",
     "mkdirp": "0.5.1",
-    "opn": "4.0.2",
```

Comment:
we use `opn` for only a very simple use case, so everything should be ok, but what were the breaking changes in 5.x and 6.x (if 6.x was more than the name change) to make sure we aren't walking into trouble?

Reply:
IMO it doesn't make much sense to go ahead with a deprecated version :thinking:

Reply:
sorry if I wasn't clear, I was just asking that since there have been two major releases since the version we currently use if there are any breaking changes we should worry about. From [the changelog](https://github.com/sindresorhus/open/compare/v4.0.2...v6.3.0), it sounds like the only material change for us is that `wait: false` is now the default.

Since we use that already, we should be ok with updating.

Reply:
:+1: